.circleci/config.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - run:
           name: Perform benchmark
           command:  |
-            aws cloudwatch put-metric-data --metric-name TimeToPublished --namespace CtfPerf --value $(node run.js) --unit Milliseconds --timestamp $(date -u +"%Y-%m-%dT%H:%M:%SZ") --region us-east-1
+            aws cloudwatch put-metric-data --metric-name TimeToPublished --namespace CtfPerf --value $(node bin/test-entry-update.js) --unit Milliseconds --timestamp $(date -u +"%Y-%m-%dT%H:%M:%SZ") --region us-east-1
```

Comment:
We should probably update the command to run the different tests in the `bin/` directory and forward their results to CloudWatch with the proper tagging.

Reply:
yes - I need some help on that